pkg/agent/openflow/pipeline.go

```diff
@@ -64,6 +64,56 @@ var (
 	ofPortRegRange = binding.Range{0, 31}
 )
 
+type flowTableDescription struct {
+	Number binding.TableIDType
+	Name   string
+}
+
+var (
+	FlowTables = []struct {
```

Comment:
What about adding the table name into `github.com/vmware-tanzu/antrea/ovs/openflow/ofTable.go as a field? We can expand interafce Table to support query table name, or get TableIDType by table name.

 According to current implementaion, agent has to maintain two slices: client.pipeline(), and FlowTables. Once we need to introduce new table, we have to change two places.

Reply:
Yes, I thought about that. But antctl also needs to know the names. If we add name in ofTable, then even antctl needs to initialize all tables.